CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 
+- Is vinyl broken check in tests (#103).
```

Comment:
At least, please, qoute 'is vinyl broken'. Otherwise it is hard to parse against the English grammar.

Reply:
No problems, I've rewrote the message:
```
### Fixed

- Incorrect check of the Tarantool version in the tests to determine a bug in
  the vinyl engine that breaks the tests (#103).
```